downstreamadapter/dispatchermanager/helper.go

```diff
@@ -196,6 +209,24 @@ func (h *SchedulerDispatcherRequestHandler) Handle(dispatcherManager *Dispatcher
 			log.Warn("scheduleDispatcherRequest is nil, skip")
 			continue
 		}
+		if req.Config == nil {
+			log.Warn("scheduleDispatcherRequest config is nil, skip")
+			continue
+		}
+		operatorKey := schedulerDispatcherKey(req.ScheduleDispatcherRequest)
+		if operatorKey == "" {
+			log.Warn("scheduleDispatcherRequest has no valid operator key, skip")
+			continue
+		}
+		// If there is already an operator for the span, skip this request.
+		_, exists := dispatcherManager.currentOperatorMap.Load(operatorKey)
```

Comment:
Consider this scenario: a dispatcher is performing a merge operation, but before it's finished, a drop table DDL causes the dispatcher to be removed. In this situation, could it happen that the dispatcher holding the operator has now received a new operator?

Reply:
when will these continue happens? It will make the create or remove action skipped.  Is there no correctness issue here?